Holovibes/includes/API.hh

```diff
@@ -660,6 +660,18 @@ bool get_contrast_auto_refresh();
  */
 void disable_convolution();
 
+/**
+ * \brief Loads a convolution matrix from a file
+ *
+ * This function is a tool / util supposed to be called by other functions
+ *
+ * \param file The name of the file to load the matrix from. NOT A FULL PATH
+ * \param convo_matrix Where to store the read matrix
+ *
+ * \throws When the matrix cannot be loaded
```

Comment:
add the exeption name 
cf wiki 
` \throw         {exception_name} {description on when}`